xs/src/libslic3r/SimplePrint.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef slic3r_SimplePrint_hpp_
+#define slic3r_SimplePrint_hpp_
+
+#include "libslic3r.h"
+#include "Point.hpp"
+#include "Print.hpp"
+
+namespace Slic3r {
+
+class SimplePrint {
+    public:
+    bool center{true};
+    std::function<void(int, const std::string&)> status_cb {nullptr};
```

Comment:
I think this can be tested similar to how I test some of the other callback methods via a lambda that sets a value to true/false as expected.